spec/ADAS/ADAS.vspec

```diff
@@ -131,6 +143,18 @@ ObstacleDetection.IsError:
   type: sensor
   description: Indicates if obstacle sensor system incurred an error condition. True = Error. False = No Error.
 
+ObstacleDetection.ForwardDistance:
+  datatype: float
+  type: sensor
+  unit: m
+  description: Distance in meters to forward detected object
+
+ObstacleDetection.ForwardTimeGap:
+  datatype: int32
```

Comment:
Could be uint32, or do we intend to use negative values to represent cases like no foreseeable impact.